onnxoptimizer/test/optimizer_test.py

```diff
@@ -1385,42 +1408,67 @@ def test_fuse_consecutive_squeezes_random(self):  # type: () -> None
         s2_one_indices = [i for i, a in enumerate(s2_x_shape) if a == 1]
         s2_axes = s2_one_indices
```

Comment:
```suggestion
        s1_axes = np.array(s1_axes, dtype=np.int64)
        s2_axes = np.array(s2_one_indices, dtype=np.int64)
```

这样 1425 行的 np.array(s2_axes) 就可以删掉了